Service/ComponentenCatalogusService.php

```diff
@@ -30,17 +30,20 @@ class ComponentenCatalogusService
     private MappingService $mappingService;
     private SymfonyStyle $io;
     private Entity $organisationEntity;
+    private DeveloperOverheidService $developerOverheidService;
 
     public function __construct(
         EntityManagerInterface $entityManager,
         CallService $callService,
         SynchronizationService $synchronizationService,
-        MappingService $mappingService
+        MappingService $mappingService,
+        DeveloperOverheidService $developerOverheidService
```

Comment:
Did you do setStyle($io) for the developerOverheidService in the setStyle() function of this service?